v2/get_catalog.go

```diff
@@ -21,11 +21,32 @@ func (c *client) GetCatalog() (*CatalogResponse, error) {
 		}
 
 		if !c.APIVersion.AtLeast(Version2_13()) {
+			// remove ParameterSchemas if broker version is less than 2.13
 			for ii := range catalogResponse.Services {
 				for jj := range catalogResponse.Services[ii].Plans {
 					catalogResponse.Services[ii].Plans[jj].ParameterSchemas = nil
 				}
 			}
+		} else if !c.APIVersion.AtLeast(Version2_14()) {
```

Comment:
We should make this dependent on alpha features being enabled until openservicebrokerapi/servicebroker#392 is merged and released in the spec, so this should be `else if c.EnableAlphaFeatures {` instead.

Reply:
Shouldn't I add `!c.EnableAlphaFeatures` as an additional condition? There's a similar if statement a few lines up, which checks if the version is less than 2.13 and removes the entire `ParameterSchemas` field. AFAICT that's because the client is supposed to only return fields that are part of the specific version of the OSBAPI spec the client is configured to use.

Reply:
Nevermind. I get what you mean. Fixed it.